integration-tests/virtual-threads/vertx-event-bus-virtual-threads/src/test/java/io/quarkus/virtual/vertx/RunOnVirtualThreadTest.java

```diff
@@ -0,0 +1,35 @@
+package io.quarkus.virtual.vertx;
+
+import static org.hamcrest.Matchers.is;
+
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Timeout;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.RestAssured;
+
+@QuarkusTest
+class RunOnVirtualThreadTest {
+
+    @Test
+    @Timeout(10)
```

Comment:
Why don't we use the Awaitility API for the timeout? I mean Awaitility is used anyway so...

Reply:
Yes, the timeout are not useful here.